cmds/install.go

```diff
@@ -326,7 +326,9 @@ func downloadFunktion() (err error) {
 
 	_, err = exec.LookPath(funktion)
 	if err != nil {
-		latestVersion, err := getLatestVersionFromGitHub(fabric8io, funktionOperator)
```

Comment:
one minor nit - the function operator still needs to be downloaded via github though - its only kubectl download that needs to use this other mechanims